scripts/gitbook.js

```diff
@@ -13,6 +13,8 @@ const ROOT_LEVEL = 3
  * Format names following https://en.wikipedia.org/wiki/AP_Stylebook and https://en.wikipedia.org/wiki/APA_style styles
  */
 const CONNECTIVES = 'a an and at but by for in nor of on or so the to up yet'
+const WHITELIST = 'api css cms adr http nova sms ui'
```

Comment:
`UPPERCASE_WHITELIST`

Had to read the code to see what it was a whitelist for.